src/main/resources/locales/en-US.yml

```
@@ -0,0 +1,6 @@
+islandfly:
+  fly-outside-alert: "You are outside your island fly mode will be disabled in [timeout] seconds."
+  disable-fly: "Your fly mode has been disabled."
+  enable-fly: "Your fly mode has been enabled."
+  command:
+    description: "Allow you to fly on your island."
```

Command description shouldn't start with an uppercase:
```suggestion
    description: "allows you to fly on your island"
```